src/main/drivers/pwm_output.c

```diff
@@ -129,18 +130,73 @@ static void pwmWriteMultiShot(uint8_t index, uint16_t value)
     *motors[index].ccr = lrintf(((float)(value-1000) * MULTISHOT_20US_MULT) + MULTISHOT_5US_PW);
 }
 
-void pwmWriteMotor(uint8_t index, uint16_t value)
+#ifdef USE_DSHOT
+static void pwmWriteDigital(uint8_t index, uint16_t value)
+{
+    pwmMotorOutput_t * const motor = &motors[index];
+
+    motor->value = value;
+    motor->updateFlags |= MOTOR_UPDATE_VALUE;
+}
+
+void pwmCompleteDigitalMotorUpdate(uint8_t motorCount)
+{
+    UNUSED(motorCount);
+
+}
+
+void pwmWriteValueToDmaBuffer(uint16_t value, uint32_t *buffer, uint8_t flags)
+{
+    buffer[0]  = (value & 0x400) ? MOTOR_BIT_1 : MOTOR_BIT_0;
```

Comment:
@ledvinap @blckmn 
I added this to the current development version
https://github.com/betaflight/betaflight/commit/07dc9eaa5ee5558eacd6fe4ede65d2bc85dcc101

@blckmn The reason why I did it is because I want to test it now as it was. I know already that it works there. Maybe the unsynced isnt even needed actualy.
Just the normal pwm unsynced needs a fix for now